dominiondiamondhotel/dominiondiamondhotel-model/src/main/java/com/ontimize/dominiondiamondhotel/model/core/service/ProductService.java

```diff
@@ -58,6 +58,28 @@ public EntityResult productInsert(Map<String, Object> attrMap) throws OntimizeJE
             return er;
         }
     }
+    @Override
+    public AdvancedEntityResult productPaginationQuery(Map<?,?> keysValues, List<?> attributesValues, int pagesize, int offset, List<SQLStatementBuilder.SQLOrder> orderby) throws OntimizeJEERuntimeException {
+        Map<String,Object>filterMap= new HashMap<>();
+
+        if(keysValues.get("pricemin")!=null && keysValues.get("pricemax")!= null){
+            Double min = Double.parseDouble(String.valueOf(keysValues.get("pricemin")));
+            Double max = Double.parseDouble(String.valueOf(keysValues.get("pricemax")));
+            if(min>0 && max>min){
+                filterMap.put(SQLStatementBuilder.ExtendedSQLConditionValuesProcessor.EXPRESSION_KEY, CommonUtils.andExpression(max, min , 2));
```

Comment:
en vez de chamar directamente o andExpression podedes chamar por separado as funcions de min e max e despois o and